samples/add-features-with-contingent-values/src/main/java/com/esri/arcgismaps/sample/addfeatureswithcontingentvalues/screens/MainScreen.kt

```diff
@@ -0,0 +1,292 @@
+                    graphicsOverlays = listOf(mapViewModel.graphicsOverlay),
+                    onSingleTapConfirmed = {
+                        it.mapPoint.let { point ->
+                            mapPoint = point
+                            showBottomSheet = true
+                        }
+                    }
+                )
+
+                if (showBottomSheet) {
+                    ModalBottomSheet(
+                        modifier = Modifier.wrapContentHeight(),
+                        sheetState = bottomSheetState,
+                        onDismissRequest = {
+                            showBottomSheet = false
+                            mapViewModel.clearFeature()
+                        }
+                    ) {
+                        BottomSheetContents(
+                            mapPoint,
+                            bottomSheetState,
+                            onBottomSheetStateChange = { state: SheetState ->
+                                if (!state.isVisible) {
+                                    showBottomSheet = false
+                                    mapViewModel.clearFeature()
+                                }
+                            },
+                            featureEditState,
+                            mapViewModel::onStatusAttributeSelected,
+                            mapViewModel::onProtectionAttributeSelected,
+                            mapViewModel::onBufferSizeSelected,
+                            mapViewModel::validateContingency
+                        )
+                    }
+                }
+            }
+
+            // message dialog view model for displaying error messages
+            mapViewModel.messageDialogVM.apply {
+                if (dialogStatus) {
+                    MessageDialog(
+                        title = messageTitle,
+                        description = messageDescription,
+                        onDismissRequest = ::dismissDialog
+                    )
+                }
+            }
+        }
+    )
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun BottomSheetContents(
+    mapPoint: Point?,
+    bottomSheetState: SheetState,
+    onBottomSheetStateChange: (SheetState) -> Unit,
+    featureEditState: FeatureEditState,
+    onStatusAttributeSelected: (CodedValue) -> Unit,
+    onProtectionAttributeSelected: (CodedValue) -> Unit,
+    onBufferSizeSelected: (Int) -> Unit,
+    onApplyButtonClicked: (Point) -> Unit,
+) {
+    val coroutineScope = rememberCoroutineScope()
+    Column(
+        modifier = Modifier.padding(start = 12.dp, end = 12.dp, bottom = 12.dp),
+    ) {
+        Text(
+            text = stringResource(R.string.add_feature),
+            style = MaterialTheme.typography.titleMedium,
+            modifier = Modifier.align(Alignment.CenterHorizontally)
+        )
+
+        Spacer(Modifier.size(8.dp))
+        Text(stringResource(R.string.attributes))
```

Comment:
I've used `labelLarge` here and other places instead of `labelMedium` - that seemed a bit condensed without also changing the font size. Not sure which is better to change.